gnomad/resources/grch38/reference_data.py

```diff
@@ -43,6 +43,16 @@ def _import_clinvar(**kwargs) -> hl.Table:
     return clinvar
 
 
+def _import_dbsnp(**kwargs) -> hl.Table:
+    dbsnp = import_sites_vcf(**kwargs)
+    dbsnp = dbsnp.group_by(dbsnp.locus, dbsnp.alleles).aggregate(
+        rsid=hl.agg.collect_as_set(dbsnp.rsid)
+    )
+    dbsnp = hl.split_multi(dbsnp, permit_shuffle=True)
```

Comment:
might be helpful to add that this `permit_shuffle` is set because the dbsnp vcf has duplicate loci